mixer/cmd/build.go

```diff
@@ -57,6 +57,7 @@ type buildCmdFlags struct {
 	numFullfileWorkers int
 	numDeltaWorkers    int
 	numBundleWorkers   int
+	ignoreMissing      bool
```

Comment:
This flag should be in `buildDeltaPacksFlags`, not in `buildCmdFlags` as it is only part of the delta-packs command.